src/lib/remark-plugins/rewrite-tutorial-links/__tests__/get-is-learn-link.test.ts

```diff
@@ -0,0 +1,91 @@
+import { getIsLearnLink } from '../utils/get-is-learn-link'
+
+const testEachCase = (testCases: [string, boolean][]) => {
+	test.each(testCases)(
+		'getIsLearnLink(%p) returns %p',
+		(input: string, expectedOutput: boolean) => {
+			expect(getIsLearnLink(input)).toBe(expectedOutput)
+		}
+	)
+}
+
+describe('getIsLearnLink', () => {
+	describe('basic inputs', () => {
+		testEachCase([
+			[undefined, false],
+			[null, false],
+			['', false],
+			['not-a-learn-link', false],
+			['/', false],
+			['/not-a-learn-link', false],
+			['/not-a/learn-link', false],
+			['/not/a/learn-link', false],
+			['/not/a/learn/link', false],
+			['learn.hashicorp.com', false],
+			['learn.hashicorp.com/not-a-learn-link', false],
+			['https://waypointproject.io/', false],
+			['https://learn.hashicorp.com/', true],
+			['https://learn.hashicorp.com', true],
```

Comment:
One route we are missing is [/search](https://learn.hashicorp.com/search)

Reply:
It's probably not linked to much in the content but that's the only outlier I can see!